### PR TITLE
Added Weird Dog Dev

### PR DESCRIPTION
**Pull request by CodeFromCirc**

### CODE REVIEW

File: communities.json [hunk 1]
```diff
@@ -117,6 +117,13 @@
       "title": "Docusaurus",
       "inviteCode": "docusaurus",
       "githubUrl": "https://github.com/facebook/Docusaurus"
+    },
+    {
+      "logo": "weirddogdev.svg",
+      "title": "Weird Dog Dev",
+      "quote": "We started Weird Dog Dev on Discord, and it's still growing today."
```
Comment (MrJacz) [inline communities.json]: someone missed a comma

Comment (ohlookitsAugust) [reply communities.json]: Jacz pls

Comment (CodeFromCirc) [reply communities.json]: lemme fix

Comment (CodeFromCirc) [reply communities.json]: below

**Pull request closed without merging**